index.html

```diff
@@ -44,14 +44,14 @@
           </a>
           <ul class="site-nav-links">
             <li><a href="#hero" class="-active">Home</a></li>
-            <li><a href="#about">The Mission</a></li>
+            <li><a href="#mission">The Mission</a></li>
             <li><a href="#schedule">Schedule</a></li>
             <li><a href="#rules">Rules</a></li>
             <li><a href="#faqs">FAQs</a></li>
             <li><a href="#registration">Registration</a></li>
           </ul>
           <div class="social-nav-container">
-            <ul class="social-nav">
+            <ul class="social-nav-links">
```

Comment:
Can we update the javascript variable `socialLinks` to select `.social-nav-links a`? The hover styles are no longer working with the class name change